tpe/lib/src/Model.cc

```diff
@@ -85,35 +87,42 @@ Entity &Model::AddModel()
   return *it->second.get();
 }
 
+//////////////////////////////////////////////////
+void Model::SetCanonicalLink(const std::size_t linkId)
```

Comment:
just to be consistent with other function, we stopped using `const` for primitive arg types